package_manager/util.py

```diff
@@ -13,3 +16,45 @@ def package_to_rule(workspace_name, s):
 
 def encode_package_name(s):
     return base64.urlsafe_b64encode(s) + ".deb"
+
+DEBIAN_RELEASES = {
+    "stretch": "9",
+    "jessie": "8",
+    "wheezy": "7",
+    "squeeze": "6.0",
+    "lenny": "5.0",
+    "etch": "4.0",
+    "sarge": "3.1",
+    "woody": "3.0",
+    "potato": "2.2",
+}
+
+def generate_os_release(distro, os_release_file):
+    """ Generates an /etc/os-release like file with information about the
+    package distribution.  VERSION and VERSION_ID are left unset if the package
+    source is from an unknown debian release.
+    """
+
+    os_release = collections.OrderedDict([
+        ("PRETTY_NAME", "Distroless"),
+        ("NAME", "Debian GNU/Linux"),
```

Comment:
Have you put any thought into how we'd support an Ubuntu feed?

Reply:
I think we could (and probably should) definitely loosen the parameterization and the URL parsing of the sources list and accept arbitrary sources, although in that case we'd have to take the user at their word on what distribution and version the source list could correspond to (which is probably fine).

Then we'd still have to have some hardcoded metadata about how the ubuntu os-release looks but it could follow the same pattern.

Reply:
Yeah, this basically sounds right to me. We hardcode debian now later on which would need to chane first.